test/e2e/README.md

```diff
@@ -98,13 +153,101 @@ For example, E2E tests can be run from Velero repository roots using the command
       CLOUD_PROVIDER=kind OBJECT_STORE_PROVIDER=aws BSL_BUCKET=<BUCKET_FOR_E2E_TEST_BACKUP> BSL_PREFIX=<PREFIX_UNDER_BUCKET> CREDS_FILE=/path/to/aws-creds \
       ADDITIONAL_OBJECT_STORE_PROVIDER=azure ADDITIONAL_BSL_BUCKET=<BUCKET_FOR_AZURE_BSL> ADDITIONAL_BSL_PREFIX=<PREFIX_UNDER_BUCKET> ADDITIONAL_BSL_CONFIG=<CONFIG_FOR_AZURE_BUCKET> ADDITIONAL_CREDS_FILE=/path/to/azure-creds
     ```
-   Please refer to `velero-plugin-for-microsoft-azure` documentation for instruction to [set up permissions for Velero](https://github.com/vmware-tanzu/velero-plugin-for-microsoft-azure#set-permissions-for-velero) and to [set up azure storage account and blob container](https://github.com/vmware-tanzu/velero-plugin-for-microsoft-azure#setup-azure-storage-account-and-blob-container)
+
+Upgrade examples:
+
+1. Run Veleor upgrade tests with pre-upgrade version:
+
+    This example will run 1 upgrade tests: v1.11.0 ~ target.
+    
+    ```bash
+    make test-e2e \
+      CLOUD_PROVIDER=aws  BSL_BUCKET=<BUCKET_FOR_E2E_TEST_BACKUP> CREDS_FILE=/path/to/aws-creds \
+      UPGRADE_FROM_VELERO_VERSION=v1.11.0
+    ```
+1. Run Veleor upgrade tests with pre-upgrade version list:
+
+    This example will run 2 upgrade tests: v1.10.2 ~ target and v1.11.0 ~ target.
+    
+    ```bash
+    make test-e2e \
+      CLOUD_PROVIDER=aws  BSL_BUCKET=<BUCKET_FOR_E2E_TEST_BACKUP> CREDS_FILE=/path/to/aws-creds \
+      UPGRADE_FROM_VELERO_VERSION=v1.10.2,v1.11.0
+    ```
+
+
+
+Migration examples:
+
+1. Migration between 2 cluster of the same provider tests:
+    
+    Before running migration test, we should prepare kubeconfig file which contains config of each cluster under test, and set `DEFAULT_CLUSTER` and `STANDBY_CLUSTER` with the corresponding context value in kubeconfig file. 
```

Comment:
Is default cluster source or destination?

Reply:
Default one is source cluster. The naming here does bring some confusion, my thought was that the default cluster can run most of tests which need a single cluster, if another cluster is needed, then call it standby. Since test will start on DEFAULT cluster, so it is as role of source cluster.